tests/functional/aws-node-sdk/test/object/completeMPU.js

```diff
@@ -138,5 +140,71 @@ describe('Complete MPU', () => {
                 _completeMpuAndCheckVid(uploadId, eTag, undefined, done);
             });
         });
+
+        describe('with tags set on initiation', () => {
+            const tagKey = 'keywithtags';
+
+            taggingTests.forEach(test => {
+                it(test.it, done => {
+                    const key = encodeURIComponent(test.tag.key);
+                    const value = encodeURIComponent(test.tag.value);
```

Comment:
probably not any better readability wise but you could ( but probably wouldn't want to ) use a map + array destructure
```suggestion
                    const [key, value] = [test.tag.key, test.tag.value].map(encodeURIComponent)
```